packs/newrelic/sensors/newrelic_application_sensor.py

```diff
@@ -0,0 +1,311 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import eventlet
+import requests
+
+from flask import jsonify, request, Flask
+from six.moves import http_client
+from six.moves import urllib_parse
+from st2reactor.sensor.base import Sensor
+
```

Comment:
For now we should still include eventlet monkey patching here in case this sensor is used with StackStorm <= 0.7